<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>137</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28,138,219);
border-radius: 24px solid;</string>
      </property>
      <property name="title">
       <string notr="true"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>25</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="2" column="0">
        <widget class="QTableView" name="tableView">
         <property name="font">
          <font>
           <family>Roboto Slab</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="signInLabel_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Roboto Slab</family>
           <pointsize>19</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Каталог товаров</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>72</number>
         </property>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="styleSheet">
            <string notr="true">QComboBox {
    border: 2px solid transparent; /* Ширина и цвет границы */
    border-radius: 4px; /* Закругление углов */
    padding: 1px 18px 1px 3px; /* Отступы внутри комбо-бокса */
    background-color: #FFFFFF; /* Цвет фона */
    color: #000000; /* Цвет текста */
}
QComboBox QAbstractItemView {
    border: 2px solid transparent; /* Граница выпадающего списка */
    background-color: #FFFFFF; /* Белый фон */
}

</string>
           </property>
           <item>
            <property name="text">
             <string>Выберите категорию</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Протеин</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Коллаген</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Креатин</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Аминокислоты</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Гейнеры</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="styleSheet">
            <string notr="true">QComboBox {
    border: 2px solid transparent; /* Ширина и цвет границы */
    border-radius: 4px; /* Закругление углов */
    padding: 1px 18px 1px 3px; /* Отступы внутри комбо-бокса */
    background-color: #FFFFFF; /* Цвет фона */
    color: #000000; /* Цвет текста */
}
QComboBox QAbstractItemView {
    border: 2px solid transparent; /* Граница выпадающего списка */
    background-color: #FFFFFF; /* Белый фон */
}</string>
           </property>
           <item>
            <property name="text">
             <string>Отсортировать товары</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По убыванию цены</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По возрастанию цены</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>По алфавиту</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>128</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="personalAccButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Roboto Slab</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(28,138,219);
color: rgb(255, 255, 255);
border-radius: 18px;</string>
      </property>
      <property name="text">
       <string>Личный кабинет</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
